metaflow/main_cli.py

```diff
@@ -349,83 +348,79 @@ def sandbox(profile):
-                env_dict['METAFLOW_ECS_S3_ACCESS_IAM_ROLE'] = ecs_s3_role
-
+                env_dict['METAFLOW_BATCH_JOB_QUEUE'] =\
+                    click.prompt('\t' + cyan('METAFLOW_BATCH_JOB_QUEUE:') + 
+                      ' AWS Batch Job Queue')
+                env_dict['METAFLOW_ECS_S3_ACCESS_IAM_ROLE'] =\
+                    click.prompt('\t' + cyan('METAFLOW_ECS_S3_ACCESS_IAM_ROLE:') + 
+                      ' IAM role granting AWS Batch access to Amazon S3')
+                metaflow_batch_container_registry =\
+                    click.prompt('\t' + cyan('METAFLOW_BATCH_CONTAINER_REGISTRY:') + 
+                      yellow(' (optional)') + ' Default docker image repository for AWS Batch jobs',
+                        default='', show_default=False)
+                if metaflow_batch_container_registry:
+                    env_dict['METAFLOW_BATCH_CONTAINER_REGISTRY'] = metaflow_batch_container_registry
+                metaflow_batch_container_image =\
+                    click.prompt('\t' + cyan('METAFLOW_BATCH_CONTAINER_IMAGE:') + 
+                      yellow(' (optional)') + ' Default docker image for AWS Batch jobs',
+                        default='', show_default=False)
+                if metaflow_batch_container_image:
+                    env_dict['METAFLOW_BATCH_CONTAINER_IMAGE'] = metaflow_batch_container_image
         # Metadata service configuration.
-        use_metadata = click.confirm('\nDo you want to use a (remote) metadata '
-                                     'service?', default=True, abort=False)
+        use_metadata = click.confirm('\nConfigure Metadata Service as default metadata provider?', 
+            default=True, abort=False)
         if use_metadata:
-            echo('\tMetadata service', fg='cyan')
-            service_url = click.prompt('\tPlease enter the URL for your '
-                                       'metadata service')
             env_dict['METAFLOW_DEFAULT_METADATA'] = 'service'
-            env_dict['METAFLOW_SERVICE_URL'] = service_url
-
-        # Conda (on S3) configuration.
-        if use_s3:
-            use_conda = click.confirm('\nDo you want to use conda for '
-                                      'dependency management?',
-                                      default=True, abort=False)
-            if use_conda:
-                echo('\tConda on AWS S3', fg='cyan')
-                default_val =\
-                    '%s/conda' % env_dict['METAFLOW_DATASTORE_SYSROOT_S3']
-                package_s3root = \
-                    click.prompt('\tPlease enter the bucket prefix for storing '
-                                 'conda packages',
-                                 default=default_val)
-                env_dict['METAFLOW_CONDA_PACKAGE_S3ROOT'] = package_s3root
+            env_dict['METAFLOW_SERVICE_URL'] =\
+                click.prompt('\t' + cyan('METAFLOW_SERVICE_URL:') +
+                  ' URL for Metadata Service (Open to Public Access)')
```

Comment:
Could you double check if sentence case or lower case - which looks better - for the info in ()?

Reply:
I went with the sentence case for continuity here.